core/process/data.loader.php

```diff
@@ -393,36 +393,6 @@
 else {
 	$home = new stdClass();
 
-	// Right now
```

Comment:
why did you remove this part?

Reply:
As far as i can tell this is unused right?

Reply:
Okay. It seems so. 👍